### PR TITLE
Filter implemented for ListedVolumes.

### PR DESCRIPTION
**Pull request by sbespalov**
For https://github.com/amihaiemil/docker-java-api/issues/214:
- `ListedVolumes` filter implemented;
- `UncheckedUriBuilder` improvements;

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#234](https://github.com/amihaiemil/docker-java-api/issues/234) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#234](https://github.com/amihaiemil/docker-java-api/issues/234) is assigned to @bkuzmic[/z](https://www.0crat.com/u/bkuzmic), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/656f92d6-7a70-46f8-85f9-aad819596c81); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by bkuzmic** [comment]
@rultor merge it

File: src/main/java/com/amihaiemil/docker/RtUriBuilder.java [hunk 1]
```diff
@@ -27,37 +27,68 @@
...
         return this;
     }
 
+    /**
+     * Adds a JSON encoded `filters` parameter.
+     * 
+     * @param filters Value of the filters.
+     * @return A {@link RtUriBuilder} instance.
+     */
+    public RtUriBuilder addFilters(
```
[the remark this answers is not in the document]
Comment (bkuzmic) [reply src/main/java/com/amihaiemil/docker/RtUriBuilder.java]: @sbespalov If you add the ticket as a Github issue directly, you'll earn 15 points. If you create a puzzle, you won't.

**Comment by bkuzmic** [comment]
@rultor merge it

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
Job was finished in 29 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#234](https://github.com/amihaiemil/docker-java-api/issues/234) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @bkuzmic[/z](https://www.0crat.com/u/bkuzmic)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**